Gemfile

```diff
@@ -15,6 +15,7 @@ group :development, :test do
   gem "awesome_print"
   gem "dotenv-rails"
   gem "factory_bot_rails"
+  gem "ffaker"
```

Comment:
Is the gem supposed to be "faker"?

Reply:
No, it's another one (or the same?!?!) :)